Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,50 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+operational direction of the Node.js Technical Steering Committee (TSC).
+
+## Organization Roles
+
+Individual users within a GitHub organization may be granted one of three
+distinct roles: Member, Owner, or Billing manager. The permissions granted
+to each role are documented in the GitHub documentated [Permission levels for
+an organization](https://help.github.com/articles/permission-levels-for-an-organization/).
+
+### Billing manager
+
+All Billing managers for the Node.js Foundation GitHub Organization shall
+be determined by the Node.js Foundation Board of Directors with notification
+given to the TSC.
+
+### Owners
+
+All voting members of the Node.js TSC shall have Owner permissions within the
+Node.js Foundation GitHub Organization. Additional users may be granted Owner
+permissions at the discretion of the TSC through normal TSC motion and simple
+majority vote.
+
+### Members
+
+GitHub users may be granted Member status in the Node.js Foundation GitHub
+Organization by any Owner.
+
+## Repositories
+
+Any organization member may create new repositories within the Node.js
+Foundation GitHub Organization. Any such repository is considered to be a
+project under the ownership of the Node.js Foundation, and thereby subject
+to the Intellectual Property and Governance policies of the Foundation under
+the operational direction of the TSC.
+
+No repository may be transfered into the Node.js Foundation GitHub Organization
+without standard TSC motion and simple majority vote.
+
+No repository may be deleted or transfered out of the Node.js Foundation GitHub
+Organization without standard TSC motion and simple majority vote. In certain
+cases, Node.js Foundation Board of Director approval may also be required.
+
+All repositories must have a README that clearly identifies the purpose of the
+repository, and governance documentation that details how the repository is
+managed.
+ 
```

Comment:
I think we should also be explicit here that it needs to have a license covering the content in the repo and that the license should be in the 'Ok' list unless there is approval from the TSC prior to the repo being created/pulled in.